yarn.lock

```diff
@@ -17004,16 +17004,16 @@ tslib@2.1.0, tslib@^2.0.3:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-2.1.0.tgz#da60860f1c2ecaa5703ab7d39bc05b6bf988b97a"
   integrity sha512-hcVC3wYEziELGGmEEXue7D75zbwIIVUMWAVbHItGPx0ziyXxrOMQx4rQEVEV45Ut/1IotuEvwqPopzIOkDMf0A==
 
+tslib@2.2.0:
+  version "2.2.0"
+  resolved "https://registry.yarnpkg.com/tslib/-/tslib-2.2.0.tgz#fb2c475977e35e241311ede2693cee1ec6698f5c"
+  integrity sha512-gS9GVHRU+RGn5KQM2rllAlR3dU6m7AcpJKdtH8gFvQiC4Otgk98XnmMU+nZenHt/+VhnBPWwgrJsyrdcw6i23w==
+
 tslib@^1.8.1, tslib@^1.9.0:
```

Comment:
Where does this one come from? `yarn why`?

Reply:
tslib contains helper functions for TypeScript.

One of the our child libraries has this in `peerDependencies`, so our project must also have this in `**dependencies**` for everything to install correctly.

Reply:
It is worth considering whether this child library is needed if it is only used in one place. It might be worth getting rid of this dependency (asvae-executors).

I encourage @asvae  to consider this issue.

Reply:
In my opinion, tslib is actually a dependency, instead of peer dependency. This library is used in `asva-executors and it actually requires ts-lib as external in its rollup config. The easiest way is simply declare `tslib` as vuestic-ui dependency instead of updating this package.

Reply:
We can make do without `asva-executors` I think, if some of it's functionality is needed it's easier to support that in context of vuestic-ui.